components/builder-admin/habitat/plan.sh

```diff
@@ -1,16 +1,25 @@
 pkg_name=hab-builder-admin
 pkg_origin=core
-pkg_version=$(cat "$PLAN_CONTEXT/../../../VERSION-BLDR")
+pkg_version=undefined
 pkg_maintainer="Jamie Winsor <reset@chef.io>"
 pkg_license=('Apache-2.0')
 pkg_source=nosuchfile.tar.gz
 pkg_bin_dirs=(bin)
 pkg_deps=(core/glibc core/openssl core/gcc-libs core/zeromq core/libsodium core/libarchive)
 pkg_build_deps=(core/protobuf core/protobuf-rust core/coreutils core/cacerts
-  core/rust core/gcc core/pkg-config)
+  core/rust core/gcc core/git core/pkg-config)
 bin="bldr-admin"
 pkg_svc_run="$bin start -c ${pkg_svc_path}/config.toml"
 
+do_verify() {
+  pushd $PLAN_CONTEXT/../../.. > /dev/null
+  pkg_version=`git rev-list master --count`
+  pkg_dirname="${pkg_name}-${pkg_version}"
```

Comment:
I had to re-define this variable and the two below because they are statically set before these functions are run.

/cc @fnichol we may want to consider making these functions instead of variables so we can lazily load things like the pkg_version in a function?